README.md

```diff
@@ -9,13 +9,15 @@ The app is designed for everyone, from beginners to experienced entrepreneurs an
 
 ## Features
 - **Practice your speech skills**: The user has 3 different practice modes: Sales pitch, public speaking, and interview. The last one works in real-time, meaning the interview questions can change depending on the user's answers thanks to the Chat-GPT API we use.
-- **Real-Time Analysis**: Provides real-time feedback on speech delivery, including aspects such as the emotions that can be derived from the words given as a response (not the tone) as well as what points the user can work on better and how good the response to an interview question/the speech given was.
+- **Real-Time Analysis**: Provides real-time feedback on speech delivery, including aspects such as the emotions that can be derived from the words given as a response as well as what points the user can work on better and how good the response to an interview question/the speech given was.
 - **Friends**: Users can have friends and see how they performed on the same interviews, giving them a sense of competition and a drive to work more on their public speaking skills.
```

Comment:
Can we see the friends profile ?

Reply:
What I mean by that is that the text "Users can have friends and see how they performed on the same interviews" is not correct.